ns_pipelines/participant_demographics/clean.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
```

Comment:
Should we clean up the namespace of each pipeline in `__init__.py`

That we, we can just import `from ns_pipeline.participant_demographics import ParticipantDemographics`?
Perhaps we can even go further and just do `from ns_pipeline.pipelines import ParticipantDemographics`. 

Looks like you do this with `word_count`

Reply:
Perhaps we could even get rid of `run,py` and define the main classes directly in `__init__.py`